docs/json-encoding.md

```diff
@@ -0,0 +1,67 @@
+# JSON Encoding
+
+In this repository, `.json` files contain a JSON encoding of TOML data, as described below.
+
+## Description
+
+TOML tables encode to JSON Objects. TOML table arrays encode to JSON Array of Objects. TOML values are encoded as a JSON Object with the following structure:
+
+```json
+{
+  "type": "<type>",
+  "value": <value>
+}
+```
+
+`<type>` is one of:
+
+- `string`
+- `integer`
+- `float`
+- `boolean`
+- `offset datetime`
+- `local datetime`
+- `local date`
+- `local time`
+- `array`
+
+When `<type>` is an `array`, the `<value>` is a JSON array containing equivalent JSON encoding of the values of the TOML array. In all other cases, `<value>` is a JSON String.
+
+Note that the only JSON values ever used are Objects, Arrays and Strings.
+
+## Example
+
+```toml
+best-day-ever = 1987-07-05T17:45:00Z
+
+[number-theory]
+boring = false
+perfection = [6, 28, 496]
+```
+
+The JSON encoding for the above is:
+
+```json
+{
+  "best-day-ever": {"type": "datetime", "value": "1987-07-05T17:45:00Z"},
+  "number-theory": {
+    "boring": {"type": "bool", "value": "false"},
```

Comment:
Per the allowed types, should be: `"boring": {"type": "boolean", "value": "false"},`